packages/schema/README.md

```diff
@@ -2785,6 +2785,37 @@ import * as S from "@effect/schema/Schema";
 const Email = S.pattern(/^(?!\.)(?!.*\.\.)([A-Z0-9_+-\.]*)[A-Z0-9_+-]@([A-Z0-9][A-Z0-9\-]*\.)+[A-Z]{2,}$/i);
 ```
 
+## Url
+
+Multiple environments like the Browser or Node provide a built-in `URL` class that can be used to validate URLs. Here we demonstrate how to leverage it to validate if a string is a valid URL while the returned value is still a string.
```

Comment:
this is an actual transformation rather than a validation, for example the string `"https://www.effect.website"` is transformed into `"https://www.effect.website/"` by `Schema.parse(Url)`

Reply:
good point, I will return the original

Reply:
but then you are defining a filter rather than a transformation so you can use the `filter` API